scripts/build_all.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import argparse
+import os
+import subprocess
+
+DEFAULT_REGION = 'us-west-2'
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--account')
+    parser.add_argument('--version')
+    parser.add_argument('--repo')
+    parser.add_argument('--region', default=DEFAULT_REGION)
+
+    return parser.parse_args()
+
+
+args = _parse_args()
+
+build_dir = os.path.join('docker', args.version, 'final')
+
+# Run docker-login so we can pull the cached image
+login_cmd = subprocess.check_output(
+    'aws ecr get-login --no-include-email --registry-id {}'.format(args.account).split())
```

Comment:
do we need the --region here for the special regions?

Reply:
this will only be run in the region where it's being built, which in our case is us-west-2, but I guess it doesn't hurt to add it